<launch>
	<arg name="camera" default="camera" /> <!--This is the camra name. Should be same as waht you use to launch the camera node-->
	<arg name="rgb_ns" default="rgb" /> <!--Generally this is rgb.-->
	<arg name="depth_ns" default="depth_registered" />
	<arg name="image_topic" default="image_rect_color" />
	<arg name="depth_topic" default="image_raw" />
	<arg name="camera_info" default="camera_info" />
	<arg name="bbox_topic_ns" default="people_detector" />
	<arg name="bbox_topic" default="bbox_measurements_array" />
	
    
	<arg name="nodename" default="people_tracking_kcf" />

	<node pkg="people_tracking" type="people_tracking_kcf" name="$(arg nodename)" output="screen">
		<remap from="camera" to="$(arg camera)" />
		<remap from="rgb_ns" to="$(arg rgb_ns)" />
		<remap from="depth_ns" to="$(arg depth_ns)" />
	    <remap from="image_topic" to="$(arg image_topic)" />
	    <remap from="depth_topic" to="$(arg depth_topic)" />
	    <remap from="camera_info" to="$(arg camera_info)" />
	    <remap from="bbox_topic_ns" to="$(arg bbox_topic_ns)" />
	    <remap from="bbox_topic" to="$(arg bbox_topic)" />
	</node>
</launch>